examples/with-vitest/vitest.config.ts

```diff
@@ -10,7 +10,7 @@ export default defineConfig({
     },
```

Comment:
`registerNodeLoader: true` seems non-existent on the new vitest version types. Do we have to use any equivalent? Or remove it will do?

Reply:
i've changed it to the deps.optimizer, which is recommended instead